### DISCUSSION TITLE
Release 22621.2361.58 breaks the weather widget

### DISCUSSION THREAD

**Discussion by gitanO0** [valinet/ExplorerPatcher#2261] [General]
Just updated to 22621.2361.58, on Windows Version 22H2 build 22621.2283.  (Latest versions of Explorer Patcher and Windows 11 as of 10/6/23)
When I hover over the weather widget on the taskbar it says "no internet access" then when I click it, a window comes up and says, "Unable to load Weather Information:  Make sure that the location you have entered is correct.
Verify that you are connected to the Internet."  

If I click the "reload" link, I see the correct weather for a short time and then the window goes back to the message above.
Thanks for looking into this.

**Comment by djc5166** [comment]
broken here too on 22621.2361.58.4

**Reply by Galthariel** [reply]
+1

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1334

please close this.
